packages/core/useStorage/index.ts

```diff
@@ -254,7 +254,9 @@ export function useStorage<T extends(string | number | boolean | object | null)>
 
     pauseWatch()
     try {
-      data.value = read(event)
+      const newData = read(event)
+      if (typeof newData !== 'object' || serializer.write(newData) !== serializer.write(data.value))
+        data.value = read(event)
```

Comment:
This doesn't seem to be very efficient. We better:

- Compare raw data with the serialized `data.value` (one serialization)
- And read the event when they are not equal (one parsing)

Reply:
thanks